mod_ngarn/__init__.py

```diff
@@ -21,17 +21,16 @@ def script():
 
 
 @click.command()
-def run():
+@click.option('--name', help='mod-ngarn table name.')
+@click.option('--limit', default=300, help='Number of greetings.')
+def run(name, limit):
     job_runner = JobRunner()
     loop = asyncio.get_event_loop()
-    loop.run_until_complete(job_runner.run())
+    loop.run_until_complete(job_runner.run(table=name, limit=limit))
 
 
 @click.command()
-@click.option(
-    '--name',
-    help='mod-ngarn table name.',
-)
+@click.option('--name', help='mod-ngarn table name.')
```

Comment:
If this is the name of a command line option you probably want to change it as name is just a bit too generic in that context.